components/mpas-ocean/src/analysis_members/Registry_conservation_check.xml

```diff
@@ -197,6 +208,9 @@
 		<var name="accumulatedLandIceFrazilSalinityFlux" type="real" dimensions="Time" units="kg m^-2 s^-1"
 			 description="Salinity flux from frazil to Land Ice, given to coupler. Positive into the ocean."
 		/>
+	    <var name="accumulatedSubglacialRunoffSalinityFlux" type="real" dimensions="Time" units="kg m^-2 s^-1"
+			 description="Salinity flux from subglacial runoff. Positive into the ocean."
+		/>
```

Comment:
Here and below, make sure indentation is with tabs, not spaces. (Yes, I know it's stupid...)